### ISSUE TITLE
Drop Down role is not reading as Drop down in NVDA

### ISSUE THREAD

**Issue by aniljami27** [microsoft/fluentui#14119]
### Environment Information

- **Package version(s)**: 7.115
- **Browser and OS versions**: Edge chromium & Win 10

### Describe the issue:

When using NVDA, when focus is on drop down, it's reading as Menu button. It should read as Drop down / Combo Box. 

#### Actual behavior:
 
It's reading as Menu button hen focus is on drop down in NVDA

#### Expected behavior:

It should read as either drop down / combo box

**Comment by paulgildea** [comment]
@xugao See if this is still the case with our latest version.

@smhigley - FYI

**Comment by xugao** [comment]
@aniljami27 - I cannot repro this [here](https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown). Currently, we have `Dropdown` has role set to `listbox` and `Combox` has role set to `combobox`. And both look correct to me. If you still think there is a bug, pls provide codpen with repro.

**Comment by aniljami27** [comment]
Thank you @xugao.. Tried to re-produce the issue in the given link and sharing the video for reference (it's reading the control type as Menu button). Please let me know if any additional info is needed. 
[DropDown_NVDA.zip](https://github.com/microsoft/fluentui/files/4951591/DropDown_NVDA.zip)

**Comment by xugao** [comment]
@aniljami27 - this is by design. if you check out aria example [here](https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html) - you can see the dropdown (button) itself is a button and it has `listbox` popup
